lighthouse-core/test/fixtures/prefetched-script.devtoolslog.json

```diff
@@ -0,0 +1,851 @@
+[
+  {
+    "method": "Page.lifecycleEvent",
```

Comment:
would you mind running this through our `./lighthouse-core/scripts/lantern/minify-devtoolslog.js` script to prune some of the unnecessary stuff?

Reply:
did you get a chance to give that script a whirl?